### PR TITLE
Remove: ExodusMovement.Exodus version 23.5.8

### PR DESCRIPTION
**Pull request by coolplaylinbot**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/121768)

### CODE REVIEW

File: manifests/e/ExodusMovement/Exodus/23.5.8/ExodusMovement.Exodus.installer.yaml
Comment (CoolPlayLin) [inline manifests/e/ExodusMovement/Exodus/23.5.8/ExodusMovement.Exodus.installer.yaml:1]: Mistake Pull Request

File: manifests/e/ExodusMovement/Exodus/23.5.8/ExodusMovement.Exodus.locale.en-US.yaml
Comment (CoolPlayLin) [inline manifests/e/ExodusMovement/Exodus/23.5.8/ExodusMovement.Exodus.locale.en-US.yaml:1]: Mistake Pull Request

File: manifests/e/ExodusMovement/Exodus/23.5.8/ExodusMovement.Exodus.yaml
Comment (CoolPlayLin) [inline manifests/e/ExodusMovement/Exodus/23.5.8/ExodusMovement.Exodus.yaml:1]: Mistake Pull Request

**Review by CoolPlayLin — CHANGES_REQUESTED** [review]

**Comment by CoolPlayLin** [comment]
Close with Reason: Mistake REMOVE

CC @stephengillie @Trenly

**Comment by stephengillie** [comment]
> Close with Reason: Mistake REMOVE
> 
> CC @stephengillie @Trenly

**Pull request closed without merging**